src/swagger.ts

```diff
@@ -0,0 +1,757 @@
+export const swaggerOptions = {
+    swaggerOptions: {
+        oauth: {
+            clientId: process.env.AUTH0_CLIENT_ID,
+            clientSecret: process.env.AUTH0_CLIENT_SECRET,
+            additionalQueryStringParams: { audience: process.env.AUTH0_AUDIENCE }
+        }
+    }
+};
+
+export const swaggerDocument = {
+    "openapi": "3.0.3",
+    "info": {
+        "title": "Expenses API Docs",
+        "description": "Swagger for Expenses API",
+        "version": "1.0.0"
+    },
+    "externalDocs": {
+        "description": "Server Github",
+        "url": "https://github.com/ofir-zeitoun/expenses-server"
+    },
+    "servers": [
+        {
+            "url": "http://localhost:1337"
```

Comment:
should it be localhost?

Reply:
I could not find any answer about which one is the best practice
All tutorials and documentation found use localhost or an API server URL (example.com), and not IP